Makefile

```diff
@@ -151,8 +142,6 @@ cover: get-libs
 
 tag-and-build:
```

Comment:
Is this target even needed any more? The GH release action does not invoke it and invoking it manually might interfere with GH.